src/rsdl.hpp

```diff
@@ -14,6 +14,7 @@
 
 struct Point {
   Point(int x, int y);
+  Point() = default;
```

Comment:
or write a default constructor to call `Point(0, 0)`?
`default` won't assign anything to `x` and `y` as I can remember, so it'll allow creating `Point` instances with _unknown_ random values for `x` and `y`. Does we want to allow this?

Reply:
Point is supposedly a data type, not an object type. (Such concept hardly exists in C++ but still) Meaning it's somehow the same as int. You can create an integer variable and the default value would be random. Not initializing an integer and using it is allowed, although it causes a warning. 
Here having a default value does not make much sense and not having a default constructor for a data type like Point with such frequent usage adds an unnecessary complexity to the code. Not allowing default construction seems more clean overall, but it was a compromise i made for code simplicity. Although this might prove unimportant or not that useful. So we can revert this change.